core/bindings.rs

```diff
@@ -46,6 +47,12 @@ lazy_static! {
       v8::ExternalReference {
         function: decode.map_fn_to()
       },
+      v8::ExternalReference {
+        function: serialize.map_fn_to()
+      },
+      v8::ExternalReference {
+        function: deserialize.map_fn_to()
+      },
```

Comment:
These two bindings need some tests, please add a new file similar to `core/encode_decode_test.js` that exercises these bindings

Reply:
I added some tests in this PR https://github.com/denoland/deno/pull/9458.